plugin/src/main/java/io/snyk/eclipse/plugin/html/StaticPageHtmlProvider.java

```diff
@@ -148,6 +152,41 @@ public String getInitHtml() {
 				""".formatted(head, base64Image, snykWarningText);
 		return replaceCssVariables(html);
 	}
-	
-	
+
+	public String getSummaryInitHtml() {
+		String htmlContent = "";
+		try {
+			InputStream inputStream = getClass().getResourceAsStream("/ui/html/ScanSummaryInit.html");
+			if (inputStream != null) {
+				htmlContent = new String(inputStream.readAllBytes(), StandardCharsets.UTF_8);
+				inputStream.close();
```

Comment:
Try using [auto-closables](https://www.perplexity.ai/search/please-explain-usage-of-try-wi-gk923RSaRJ65ZrI3PsApNA) here, please, so we are sure to avoid file descriptor leaks.